### ISSUE TITLE
Support passing in a list of classes to 'transition-class'

### ISSUE THREAD

**Issue by sumeetattree** [peec/ember-css-transitions#25]
Support for passing multiple classes is missing. Even though `transitionClass` is an array, there is no way to pass multiple classes from the template.

This currently does not work
```hbs
{{#transition-group transition-class="slide-out fade-in"}}
...
{{/transition-group}}
```

**Comment by sumeetattree** [comment]
Okay, that was totally an oversight on my part. I looked at `transitionClasses`. Upon closer inspection those two are different.

> In any case, this could be an interesting feature.

Hope to see this implemented soon!

**Comment by miguelcobain** [comment]
https://github.com/peec/ember-css-transitions/pull/38 should address this. Right @pdud?

**Comment by pdud** [comment]
Yeap! Should be addressed! It only supports one `name=` argument but you can support multiple classes via the 6 new transition classes!

**Comment by miguelcobain** [comment]
Closed by #38. Released in `2.0.0`.
